app/src/main/java/com/sopt/clody/presentation/ui/login/LoginViewModel.kt

```
@@ -91,6 +92,9 @@ class LoginViewModel @AssistedInject constructor(
         )
     }
 
+    private suspend fun loginWithGoogle(context: Context) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)